### PR TITLE
do not show dynamic for generics

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

File: lib/src/model.dart [hunk 1]
```diff
@@ -1424,15 +1424,18 @@ class ElementType {
 
     // not TypeParameterType or Void or Union type
     if (isParameterizedType) {
-      buf.write('&lt;');
-      for (int i = 0; i < typeArguments.length; i++) {
-        if (i > 0) {
-          buf.write(', ');
+      var list = typeArguments.where((t) => t.linkedName != 'dynamic').toList();
+      if (list.isNotEmpty) {
```
Comment (sethladd) [inline lib/src/model.dart]: this might be easier if we use list.join(',')

**Comment by sethladd** [comment]
thanks!

File: lib/src/model.dart (hunk 1 above)
Comment (keertip) [inline lib/src/model.dart]: done.

**Comment by sethladd** [comment]
lgtm!

**Pull request merged**